cloudca/resource_cloudca_vpn_user.go

```diff
@@ -0,0 +1,108 @@
+package cloudca
+
+import (
+	"fmt"
+
+	"github.com/cloud-ca/go-cloudca"
+	"github.com/cloud-ca/go-cloudca/services/cloudca"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceCloudcaVpnUser() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudcaVpnUserCreate,
+		Read:   resourceCloudcaVpnUserRead,
+		Delete: resourceCloudcaVpnUserDelete,
+
```

Comment:
The same `importer` as above is missing here too.

Reply:
Ok, I will add this.  Thanks...

Reply:
+1

Reply:
+1